test/test_visualization.jl

```diff
@@ -87,8 +87,6 @@ using CairoMakie
     meshFileFormat = getMeshFileFormat(p_visu)
     setFileNames!(p_visu, meshFileFormat)
 
-    @test_nowarn updatePlot!(p_visu)
-
```

Comment:
Why did you remove this test?

Reply:
The function `updatePlot!` with only one input argument was legacy code. Internally it was never used for anything. Although now that I write this, it makes this PR a breaking change if I completely remove it. I will put it back in to avoid this.